client_plugin/utils/config/config.go

```diff
@@ -19,10 +19,27 @@ package config
 
 import (
 	"encoding/json"
+	"flag"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/natefinch/lumberjack"
+	"github.com/vmware/docker-volume-vsphere/client_plugin/utils/log_formatter"
 	"io/ioutil"
+	"os"
+	"runtime"
 )
 
 const (
+	// PhotonDriver is a vmdk plugin driver for photon platform
```

Comment:
It seems according to go check, it needs to be in the format of 
// PhotonDriver is ....
PhotonDriver = "photon"